test/controllers/organizations_controller_test.rb

```diff
@@ -0,0 +1,92 @@
+require 'test_helper'
+
+class OrganizationsControllerTest < ActionController::TestCase
+  def setup
+    @user             = users(:tobias)
+    session[:user_id] = @user.id
+  end
+
+  test '#new returns success' do
+    stub_get_json("https://api.github.com/users/#{@user.login}/orgs",
```

Comment:
I'd create a context for all the test of the same action and group them that way. Once they're grouped, you can reduce duplication in your stubs.